GRDBCipher.podspec

```diff
@@ -21,5 +21,5 @@ Pod::Spec.new do |s|
 		'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) SQLITE_HAS_CODEC=1 GRDBCIPHER=1 SQLITE_ENABLE_FTS5=1'
 	}
 	s.framework = 'Foundation'
-	s.dependency 'SQLCipher', '~> 3.4.1'
+	s.dependency 'SQLCipher', '~> 4.0.1'
```

Comment:
Please remove this change: This pull request does not require any specific SQLCipher version. This is its strength :-) We'll perform the necessary changes for SQLCipher 4 in a separate PR.

It may be inconvenient to you, but I prefer that we properly phase changes, and above all I want that the GRDB-4.0 branch, target of this PR, remains usable and consistent for all GRDB users. SQLCipher 4 is not yet there.

Reply:
No problem. 👌